lib/driver.js

```diff
@@ -93,7 +90,15 @@ class YouiDriver extends BaseDriver {
       if (this.driverShouldDoProxyCmd(cmd))
       {
         logger.debug(`Executing proxied WebDriver command '${cmd}'`);
-        return this.proxydriver.executeCommand(cmd, ...args);
+
+        // There are 2 CommandTimeout (YouiDriver and proxy)
+        // Only YouiDriver CommandTimeout is used; Proxy is disabled
+        // All proxy commands needs to reset the YouiDriver CommandTimeout
+        // Here we manually reset the YouiDriver CommandTimeout for commands that goe to proxy.
+        this.clearNewCommandTimeout;
```

Comment:
it doesn't matter except for readability, but can you add parenthesis on the end of that?
  this.clearNewCommandTimeout();
lets us know its a method.